src/docs/Ways of Working/git.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 4
+slug: /git-flow
+---
+
+# Git flow
+
+This document describes the Team's git wokflow. It is bases on the [GitHub workflow](https://docs.github.com/en/get-started/quickstart/github-flow).
+
+## Prerequisites
+
+### Note.js
+
+Install [Note.js](https://nodejs.org/en/download/)
+
+### Clone repository 
+
+- Get repository URL. From the repository page on Github, click `Code` and copy URL.
+- Clone locally. (downloads repository)
+    - **Command line** `git clone URL`
+    - **VSCode** use `clone repository`
+
+### Install Dependencies
+
+**If required** by the project, update npm dependencies locally (on your computer)
+- **Command line**: `npm instal`
+
+## Workflow
+
+### 1. Create a branch
+
+Create a branch in your repository. Use your name initials and a short, descriptive branch name enables your collaborators to see ongoing work at a glance. For example, `CE-increase-test-timeout` or `NR-add-code-of-conduct`
```

Comment:
This reads a bit like an instruction 'Create a branch in your repository'. If I was following this i might have read that sentance and tried to create the branch. But you go on to explain how to do it further down.

'First you are going to need to create a branch in your repository. To do this you need to pick a name for your branch. Use your name initials and a short, descriptive branch name enables your collaborators to see ongoing work at a glance. For example, `CE-increase-test-timeout` or `NR-add-code-of-conduct`'